### DISCUSSION TITLE
無法安裝array30IME

### DISCUSSION THREAD

**Discussion by qwertyuiopcasper** [gontera/array30#9] [Q&A]
無法安裝輸入法
![image](https://github.com/gontera/array30/assets/170515300/ef9be8a6-9e9d-487a-b4bb-60947320ee07)

曾經安裝後並解除安裝
第二次要重新安裝時無法安裝
嘗試過readme裡面的作法仍無法成功安裝

**Comment by gontera** [comment]
本站是關於行列30/行列10 鍵碼表的討論區，恕本人無法在此回應；有關輸入法軟體的疑問，請洽各該軟體官網尋求解決，或亦可在 [Facebook 的行列輸入法社群](https://www.facebook.com/groups/517104371955479)中討論。
